src/db/options.rs

```diff
@@ -117,6 +124,12 @@ pub enum ValidationAction {
     Warn,
 }
 
+#[derive(Clone, Debug, PartialEq, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct IndexOptionDefaults {
```

Comment:
the struct needs a docstring. it can be similar to the one in `CreateCollectionOptions`, while also mentioning that it should be used when creating collections.